mathics/core/atoms.py

```diff
@@ -559,10 +559,10 @@ def __hash__(self):
         return self.hash
 
     def __str__(self) -> str:
-        return base64.b64encode(self.value).decode("utf8")
+        return f"<{len(self.value)}>"
 
     def atom_to_boxes(self, f, evaluation) -> "String":
-        res = String('""' + self.__str__() + '""')
+        res = String(self.__str__())
```

Comment:
Here, we should build the return string conditionated to `f`.

Reply:
Was done in a0c9e55f